### DISCUSSION TITLE
How to create new tags?

### DISCUSSION THREAD

**Discussion by oneandonlyonebutyou** [gitpython-developers/GitPython#1237] [Q&A]
I am trying to create new tags and push it to master, yet it only create locally .. 
what would be the most stable way to do that (I am trying to automate this .. )

**Comment by Byron** [comment]
The reference talks about [how to create tags](https://gitpython.readthedocs.io/en/stable/tutorial.html#advanced-repo-usage) and [here is how to use remotes](https://gitpython.readthedocs.io/en/stable/tutorial.html#handling-remotes) to push and pull.

It should be possible to push a tag using something like `repo.remotes.origin.push(tag)` where `tag` was created with `create_tag(…)` prior.
